cli/src/commands/enrich.rs

```diff
@@ -140,7 +140,7 @@ impl SnykProvider {
             }
             true => {
                 storage = Box::new(FileSystemStorageProvider::new(
```

Comment:
So Rust has a crate that is able to "figure out where the best place for a temp dir is".  It is this one here: https://docs.rs/tempfile/latest/tempfile/fn.tempdir.html

I went looking for it because Java has it built in and Rust has it too.  I recommend using this crate here instead of assuming that we have a `/tmp` folder.

Reply:
Good idea!  Looking into that now.

Reply:
I "think" we agreed to handle this in a subsequent PR.